plugin/gcp/root.go

```diff
@@ -0,0 +1,101 @@
+// Package gcp presents a filesystem hierarchy for Google Cloud Platform resources.
+//
+// It follows https://cloud.google.com/docs/authentication/production to find your credentials:
```

Comment:
Isn't this already documented? Why state it more than once?

Reply:
Different audiences. But you're probably right that it doesn't need to be in the godocs.

Reply:
Yeah, that's one less piece of documentation to maintain.